src/opm/parser/eclipse/EclipseState/Schedule/OilVaporizationProperties.cpp

```diff
@@ -91,6 +91,20 @@ namespace Opm {
         }
     }
 
+    void OilVaporizationProperties::updateDRSDTCON(OilVaporizationProperties& ovp, const std::vector<double>& maximums, const std::vector<std::string>& options){
+        ovp.m_type = OilVaporization::DRSDTCON;
+        ovp.m_maxDRSDT = maximums;
+        for (size_t pvtRegionIdx = 0; pvtRegionIdx < options.size(); ++pvtRegionIdx) {
```

Comment:
Seems like there is an assumption here that the number of elements in `options` array is equal to the range of `pvtRegionIdx`?

Reply:
This code is just a copy of the DRSDT code. The drsdt infrastructure supports region based drsdt using DRSDTR. For DRSDTCON and DRSDT the one given element is used for all PVT regions.